lib/cloud_foundry.rb

```diff
@@ -11,6 +11,15 @@ def self.vcap_data
     end
   end
 
+  def self.app_url
+    if self.is_environment?
+      # (arbitrarily) use the shortest assigned route, if there are multiple
+      self.vcap_data['application_uris'].min_by(&:length)
+    else
+      nil
```

Comment:
I think explicitness is a great cause in general but it doesn't seem to me that this is a case where the shorter version is confusing. also, I looked around and we do not appear to be returning `nil` from `else` in other parts of this app, so it feels inconsistent to me.